README.md

```diff
@@ -23,4 +23,51 @@ Advanced Course Search Bar - A single course search bar with ElasticSearch for c
 
 ## Run Locally
 
-WIP
+Each command in this section should be run from the root directory of the repository (i.e. this directory).
+
+1. Install nodejs:
+
+[https://nodejs.org/en/](https://nodejs.org/en/)
+
+2. Scrape course pages to get course database
+
+```bash
+source buildpack-run.sh
+```
+
+You will see a list of courses printed to screen as the scraper runs. It should take a few minutes to scrape everything.
+
+Note: Try not to scrape course pages too often; you should only need to do this once.
+
+3. Install node dependencies
+
+```bash
+npm install
+```
+
+4. I had to install yarn separately for some reason.
+
+From `/client` directory:
+```bash
+npm install yarn
+```
+Then, build the client:
+
+```bash
+yarn build
+```
+This will produce a new folder in `/client` called `build`.
```

Comment:
It's probably because both the client and the server have their own set of dependencies (I definitely could've been more clear about that 😅 ) You can actually build with just npm if you cd into client and then run "npm install" and "npm run build". Maybe the documentation can include both options (yarn and npm)?